app/admin/cargos.rb

```diff
@@ -2,5 +2,57 @@
 
 ActiveAdmin.register Cargo do
   permit_params :name, :surname, :middle_name, :phone, :email, :weight, :length, :width, :height, :origins,
-                :destinations, :distance, :price, :user_id
+                :destinations, :distance, :price, :user_id, :status
+
+  form do |f|
+    f.inputs do
+      f.input :name
+      f.input :surname
+      f.input :middle_name
+      f.input :phone
+      f.input :email
+      f.input :weight
+      f.input :length
+      f.input :width
+      f.input :height
+      f.input :origins
+      f.input :destinations
+      f.input :status, as: :select, collection: @resource.aasm.states(permitted: true).map { |state| state.name.to_s }, include_blank: false
```

Comment:
This implementation always change status in update action. We can't stay in current status